HW1/P4/P4 bfs.py

```diff
@@ -0,0 +1,50 @@
+#I discussed the solution for this problem with Avery Faller.
+
+# import findspark
+# findspark.init()
+import pyspark
+import numpy as np
+
+
+def copartitioned(RDD1, RDD2):
+    "check if two RDDs are copartitioned"
+    return RDD1.partitioner == RDD2.partitioner
+
+
+def filterer(i):
```

Comment:
You have parameter i here, but you don't use it in the function.  Did you mean to use itr_acc?